app/controllers/account_controller.rb

```diff
@@ -100,7 +100,7 @@ def lost_password
         # create a new token for password recovery
         token = Token.new(user: user, action: 'recovery')
         if token.save
-          UserMailer.password_lost(token).deliver
+          UserMailer.password_lost(token).deliver_now
```

Comment:
I realize, this is the 1:1 translation but can't we use `deliver_later` instead @basiszwo, @myabc? Would this use our readily available delayed_job then?